securecookie.go

```diff
@@ -703,7 +703,7 @@ func (o *Obj) Delete(w http.ResponseWriter) error {
 		b = append(b, "; Domain="...)
 		b = append(b, o.domain...)
 	}
-	b = append(b, "; Expires=Jan 2 15:04:05 2006"...)
+	b = append(b, "; Max-Age=0; Expires=Thu, 01 Jan 1970 00:00:00 GMT"...)
```

Comment:
The date formal I use follows the specification of the RFC (https://www.rfc-editor.org/rfc/rfc6265#section-5.1.1).
The time format you use does not respect the standard. 

Regarding the addition of the Max-Age field with a value 0, the RFC species that it is ignored and the value Exipres is used instead. 

Because secure cookies are critical for the security, I decided to stick to the RFC as the standard.